Gemfile

```diff
@@ -3,7 +3,8 @@ source 'https://rubygems.org'
 gemspec
 
 group :development do
-  gem 'activesupport', '< 5',      require: false
+  gem 'activesupport', "#{RUBY_VERSION < '2.2' ? '<' : '>'} 5", require: false
```

Comment:
realistically not going to happen in this case, but comparing versions as strings otherwise could lead to undesirable results (ex. `'10.1' < '2.2' == true`)